Source/Relativity.Test.Helpers/Relativity.Tests.Helpers.Tests.Unit/ArtifactHelpers/FieldsTest.cs

```diff
@@ -18,65 +18,70 @@ namespace Relativity.Tests.Helpers.Tests.Unit.ArtifactHelpers
 	public class FieldsTest
 	{
 		private IHttpRequestHelper _httpRequestHelper;
-		private readonly int _workspaceId = 1017834;
+		private readonly int _workspaceId = 1234567;
+		private Mock<IHttpRequestHelper> _httpRequestHelperMocked;
+		private IFieldsHelper Sut;
 
-		private IFields _fieldsHelper;
-
-		[OneTimeSetUp]
+		[SetUp]
 		public void SetUp()
 		{
-
+			_httpRequestHelperMocked = new Mock<IHttpRequestHelper>();
+			_httpRequestHelper = _httpRequestHelperMocked.Object;
 		}
 
-		[OneTimeTearDown]
+		[TearDown]
 		public void Teardown()
 		{
-
+			_httpRequestHelper = null;
+			_httpRequestHelperMocked = null;
+			Sut = null;
 		}
 
 		[Test]
 		public void GetFieldArtifactId()
 		{
 			//Setup
-			string _responseJson = "{\"ArtifactId\": 1037705}";
-			var httpRequestHelperMocked = new Mock<IHttpRequestHelper>();
-			httpRequestHelperMocked.Setup(x => x.SendPostRequest(It.IsAny<RequestModel>(), It.IsAny<string>())).Returns(_responseJson);
-			_httpRequestHelper = httpRequestHelperMocked.Object;
+			string _responseJson = "{\"ArtifactId\": 1223344}";
+			_httpRequestHelperMocked.Setup(x => x.SendPostRequest(It.IsAny<BaseRequestModel>(), It.IsAny<string>())).Returns(_responseJson);
+			
 
 			//Arrange
-			string fieldName = "Production::Sort Order";
-			int _productionSortOrderFieldId = 1037705;
+			string fieldName = "TestField";
+			int fieldId = 1223344;
 
-			_fieldsHelper = new Fields(_httpRequestHelper);
+			Sut = new FieldsHelper(_httpRequestHelper);
 
 			//act
-			var fieldArtifactId = _fieldsHelper.GetFieldArtifactId(fieldName, _workspaceId);
+			var fieldArtifactId = Sut.GetFieldArtifactId(fieldName, _workspaceId);
 
 			//assert
-			Assert.AreEqual(fieldArtifactId, _productionSortOrderFieldId);
+			Assert.AreEqual(fieldArtifactId, fieldId);
 
+			//Verify
+			_httpRequestHelperMocked.Verify(x => x.SendPostRequest(It.IsAny<BaseRequestModel>(), It.IsAny<string>()), Times.Exactly(1));
 		}
 
 		[Test]
 		public void GetFieldCount()
 		{
 			//setup
 			string _responseJson = "{\"Count\": 1}";
-			var httpRequestHelperMocked = new Mock<IHttpRequestHelper>();
-			httpRequestHelperMocked.Setup(x => x.SendPostRequest(It.IsAny<RequestModel>(), It.IsAny<string>())).Returns(_responseJson);
-			_httpRequestHelper = httpRequestHelperMocked.Object;
+			_httpRequestHelperMocked.Setup(x => x.SendPostRequest(It.IsAny<BaseRequestModel>(), It.IsAny<string>())).Returns(_responseJson);
 
 			//arrange
 			int _count = 1;
-			const int fieldArtifactId = 1037705;
+			const int fieldArtifactId = 1223344;
 
-			_fieldsHelper = new Fields(_httpRequestHelper);
+			Sut = new FieldsHelper(_httpRequestHelper);
```

Comment:
i think this line can be moved to the Setup as it's a common step in both the tests.